frontend/www/js_templates/modify_l2vpn.js

```diff
@@ -163,6 +163,11 @@ document.addEventListener('DOMContentLoaded', async function() {
   state = new GlobalState();
   state.selectCircuit(id);
 
+  if (state.circuit == null){
+    document.getElementById("connection_error").style.display = "block";
+    document.getElementById("circuit2").style.display = "none";
```

Comment:
`circuit2` or `circuit`?

Reply:
This works for both L2 and L3 connections. "circuit2" is the name of the circuit information div that is displayed on the page for L2 connections. I had to update the L2 connections error case as it would show the error on valid circuit IDs.